src/ServiceControl.Monitoring/Settings.cs

```diff
@@ -120,7 +120,7 @@ static Type LoadTransportCustomizationType(string configuredTransportCustomizati
             {"ServiceControl.Transports.RabbitMQ.DirectRoutingTopologyRabbitMQTransport, ServiceControl.Transports.RabbitMQ", "ServiceControl.Transports.RabbitMQ.RabbitMQDirectRoutingTransportCustomization, ServiceControl.Transports.RabbitMQ"},
 
             //ASQ
-            {"NServiceBus.AzureStorageQueueTransport, NServiceBus.Azure.Transports.WindowsAzureStorageQueues", "ServiceControl.Transports.ASQ.ASQTransportCustomization, ServiceControl.Transports.ASQ"},
```

Comment:
Shouldn't this be a new entry? In theory we might have legacy installations with either the old or the new values in this diff.

Reply:
The transport is the same transport, what's changed is the package name. Everything else remains the same.

I'm not quite following how this part works in SC. Is this documented somewhere?

Reply:
I don't think it's documented.

This sets up a mapping from the old config value to the real transport type to be used. So it used to be that if the config contained `NServiceBus.AzureStorageQueueTransport, NServiceBus.Azure.Transports.WindowsAzureStorageQueues` then we would use `ServiceControl.Transports.ASQ.ASQTransportCustomization, ServiceControl.Transports.ASQ` instead. 

We want that to still be true, don't we?

Reply:
@mikeminutillo, I'm not sure that's accurate. When a new instance of SC/SC.Audit/SC.Monitoring is installed, it adds the following transport identifier to the configuration file:
```xml
<add key="ServiceControl/TransportType" value="ServiceControl.Transports.ASQ.ASQTransportCustomization, ServiceControl.Transports.ASQ" />
```

Given that the Core transport identifier (`"NServiceBus.AzureStorageQueueTransport, NServiceBus.Transport.AzureStorageQueues"`) has never been used by any old SC* installation, it will never appear in the existing configuration files and therefore does not need to be mapped to the core-provide transport identifier. As a result, the mappings do **not** need to be added. Would it be possible for you to sync with @danielmarbach, who's also involved in this change, to discuss and to an agreement wherever it's not needed or needed?

Reply:
You are correct!

There should be no legacy systems using this new transport, so there is no need to add a mapping here at all. This code should be untouched.